contracts/allowlist/ValidatorSmartContractAllowList.sol

```diff
@@ -0,0 +1,165 @@
+// Implementation of a contract to select validators using an allowlist
+
+pragma solidity >=0.5.0;
+
+import "../ValidatorSmartContractInterface.sol";
+
+contract ValidatorSmartContractAllowList is ValidatorSmartContractInterface {
+
+    event AllowedAccount(
+        address account,
```

Comment:
I would recommend declared all `address`es in our events as `indexed address` to allow efficient retrieval of subsets of events.